designsafe/apps/api/users/utils.py

```diff
@@ -104,7 +152,7 @@ def _get_latest_allocations(username):
     Creates or updates allocations cache for a given user and returns new allocations
     """
     user = get_user_model().objects.get(username=username)
-    allocations = _get_tas_allocations(username)
+    allocations = get_detailed_tas_allocations(username)
```

Comment:
Since this is the only place that `_get_tas_allocations` is being called, I'd recommend either deleting the `_get_tas_allocations` function or rename `get_detailed_tas_allocations` to `_get_tas_allocations` instead.